tokenizers-cli/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "tokenizers-cli"
+authors = ["Anthony MOI <m.anthony.moi@gmail.com>", "Nicolas Patry <patry.nicolas@protonmail.com>"]
+edition.workspace = true
+version.workspace = true
+homepage.workspace = true
+repository.workspace = true
+documentation.workspace = true
+license.workspace = true
+keywords.workspace = true
+readme = "./README.md"
+description.workspace = true
+
+[[bin]]
+name = "tokenctl"
```

Comment:
was previously just `cli` but if this were published publicly, it would need to be more descriptive. open to feedback on what name to use for the executable